pkg/server/commands/reverseexpand/reverse_expand_test.go

```diff
@@ -152,8 +152,7 @@ type document
 		res := <-resultChan
 		t.Logf("after receive one result")
 		cancelFunc()
-		t.Logf("after send cancellation")
-		require.NotNil(t, res.Object)
+		t.Logf("received object %s ", res.Object)
```

Comment:
should we throw panic if res.Object is nil?  Or change the function to return err if res.Object is nil?

Reply:
👍 added a panic

Reply:
Why not stay consistent with the wait group pattern established below?

Reply:
Added some comments to the test

Reply:
`require.NotEmpty` will have the same effect as the panic, it will stop the test if the object is empty, let's avoid usages of panic as much as possible across both test files and main code base.